pyprof/tests/L0/run_pyprof_data/test_pyprof_data.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```

Comment:
if this change is unrelated to this PR, can it be removed and saved for a separate PR?

Reply:
It is already